vm/src/stdlib/itertools.rs

```diff
@@ -953,6 +953,87 @@ impl PyItertoolsCombinations {
     }
 }
 
+#[pyclass]
+#[derive(Debug)]
+struct PyItertoolsZiplongest {
+    iterators: Vec<PyObjectRef>,
+    fillvalue: PyObjectRef,
+    numactive: RefCell<usize>,
```

Comment:
This can just be a `Cell<usize>`. `Cell` is better for primitive data types that can just be copied. Then, later, you can just do `let numactive = self.numactive.get();` and `self.numactive.set(new_numactive);`